Makefile

```
@@ -26,8 +26,10 @@ build: export SPDK_GIT_REPO != git -C spdk remote get-url origin
 build: export SPDK_GIT_BRANCH != git -C spdk name-rev --name-only HEAD
 build: export SPDK_GIT_COMMIT != git rev-parse HEAD:spdk
 build: export BUILD_DATE != date -u +"%Y-%m-%dT%H:%M:%SZ"
+build: export NVMEOF_VERSION != git describe --tags --abbrev=0
```

Instead of defining this variable per target, why not doing it globally?:
```suggestion
export NVMEOF_VERSION != git describe --tags --abbrev=0
```
If that's an overkill (because not all `make` targets require the `NVMEOF_VERSION`), we could simply selecto those at once:
```suggestion
build up demo run logs push: export NVMEOF_VERSION != git describe --tags --abbrev=0
```

---

done